### ISSUE TITLE
Draft Elicitation Questions

### ISSUE THREAD

**Issue by aykutkantas** [aykutkantas/coding-for-me#3]
Draft Elicitation Questions have been prepared

1.	What was your job at the fast-food restaurant, and what did it entail?
2.	Can you describe a difficult situation you faced while working there and how you dealt with it?
3.	How did your experience at the restaurant shape your views on customer service?
4.	What did you learn about teamwork while working there, and how did you apply those lessons in other areas of your life?
5.	Did you have any mentors or role models at the restaurant who influenced your work ethic or career aspirations? If so, can you tell me more about them?
6.	What was the most challenging aspect of working at the restaurant, and how did you overcome it?
7.	Looking back, do you think working at the restaurant was a valuable experience? If so, why? If not, why not?
8.	Did you form any lasting friendships or relationships with your coworkers while working at the restaurant? If so, can you tell me more about them?

**Comment by aykutkantas** [comment]
Some questions asked to gain more information. 

1.	Can you tell me more about the specific incident with the difficult customer at the fast-food restaurant? What was the customer upset about, and how did you resolve the issue?
2.	How long did you work at the fast-food restaurant, and did you have any other jobs before or after that experience?
3.	In what ways did working at the restaurant impact your personal and professional development? Did it teach you any skills or qualities that you continue to use in your current career?
4.	Were there any particularly positive or negative experiences you had while working at the restaurant that stand out in your memory?
5.	Did you receive any training or guidance from management or colleagues while working at the restaurant? If so, can you tell me more about that experience?
6.	How did your experience at the fast-food restaurant shape your views on customer service, and do you believe those lessons are still relevant today?
7.	Did you have any other responsibilities or duties beyond customer service while working at the restaurant? If so, what were they?
8.	How did your coworkers or colleagues influence your experience working at the restaurant, and do you still keep in touch with any of them today?

**Comment by aykutkantas** [comment]
In addition to my friend's story, some more information have been gained with additional questions. 

1.	Customer was upset about a mistake with his order and long wait times. To Resolve this issue, I benefited from active listening, empathy, and finding a solution that meets the customer's needs while following the restaurant's policies.
2.	I work part-time while in summer. Additionally, after this experience I had one more part time job but it was not related to fast food, it was selling sim cards and phone accessories at a Vodafone shop.
3.	Working at a fast-food restaurant can teach various skills. I can say that I have learned much about customer service, time management, teamwork, and multitasking. Additionally, it can teach qualities such as adaptability, resilience, and a strong work ethic. I have benefited from these skills and qualities for my second job also.
4.	My positive experiences are generally regarding relationships with my colleagues and customers, learning new skills, and developing a sense of pride in providing a valuable service. On the other hand, to cite some examples related to my negative experiences, I can say dealing with difficult customers, working in a fast-paced and high-stress environment, and experiencing long working hours.
5.	We have taken some training and guidance from management and colleagues. These helped us to learn how to provide excellent customer service, operate equipment, and handle food safely. Especially, guidance from experienced colleagues have ensured to learn the best practices for efficiency and teamwork.
6.	Working in the fast-food restaurant has shaped my views on customer service by emphasizing the importance of active listening, empathy, and finding solutions that meet the customer's needs. I consider that these lessons are still relevant today in any industry that involves working with people.
7.	In addition to customer service, I have some other responsibilities such as food preparation, cleaning, and maintaining equipment. 
8.	Positive relationships between my colleagues provided me always support, guidance, and a sense of community. Thus, we worked in an environment more enjoyable. Today, we have maintained relationships with my some of coworkers, whether through social media or in-person meetings.
